src/adapters/lokijs/worker/lokiExtensions.js

```diff
@@ -60,13 +58,7 @@ async function getLokiAdapter(options: LokiAdapterOptions): mixed {
         ? require('lokijs/src/incremental-indexeddb-adapter')
         : require('lokijs/src/incremental-indexeddb-adapter')
       // $FlowFixMe
-      return new IncrementalIDBAdapter({
-        onversionchange: onIndexedDBVersionChange,
```

Comment:
Oh I see, this has always been lowercase

Reply:
yes. ugly, but not worth changing IMO. `onversionchange` follows the DOM convention